libraries/default.rb

```diff
@@ -2,14 +2,23 @@
 
 module Hops
   module Helpers
-    def template_ssl_server(generate_jwt = true)
-      if (generate_jwt and (node['hops']['rmappsecurity']['jwt']['enabled'].eql?("true") or node['hops']['tls']['enabled'].eql?("true")))
+    def template_ssl_server(create_api_key = true, requester)
+      if (create_api_key and (node['hops']['rmappsecurity']['jwt']['enabled'].eql?("true") or node['hops']['tls']['enabled'].eql?("true")))
```

Comment:
What is the ['jwt']['enabled'] ?

Reply:
A flag for ResourceManager to generate application JWTs on submission